src/EntityFramework.Sqlite.Design/SqliteTableSelectionSetExtensions.cs

```
@@ -8,14 +8,38 @@ namespace Microsoft.Data.Entity.Scaffolding
 {
     internal static class SqliteTableSelectionSetExtensions
     {
+        /// <summary>
+        /// Tests whether the table is allowed by the <see cref="TableSelectionSet" /> and
+        /// updates the <see cref="TableSelectionSet" />'s <see cref="TableSelectionSet.Selection" />(s)
+        /// to mark that they have been satisfied.
+        /// </summary>
+        /// <param name="tableSelectionSet"> the <see cref="TableSelectionSet" /> to test </param>
+        /// <param name="tableName"> name of the database table to check </param>
+        /// <returns> whether or not the table is allowed </returns>
         public static bool Allows(this TableSelectionSet tableSet, string tableName)
         {
-            if ((tableSet == null)
-                || (tableSet.Tables.Count == 0))
+            // mark all schema selections as satisfied as sqlite ignores them
```

@natemcmaster @rowanmiller I'd like some advice on this. At the moment I've coded it so that any `-s` entries the user makes for SQLite are basically silently ignored - they will not cause a warning saying that the selection is not satisfied, nor will they cause any warning that using a `-s` flag under SQLite is basically meaningless. I can continue to take this approach or I can:

A) take out this section - which means if the user enters any `-s` flags all of them will raise warnings saying they are unsatisfied.
B) check for `-s` flags much earlier in the process - the question then is do I a) error - i.e. issue an error message and stop the process; or b) warn that `-s` flag is meaningless but continue, and when I continue do I also raise the warning messages that the `-s` flags are unsatisfied?